WordPress/Classes/ViewRelated/Tools/PromptViewController.swift

```diff
@@ -0,0 +1,123 @@
+import Foundation
+
+
+/// Protocol that defines the methods required by PromptViewController's Children(s).
+///
+@objc
+protocol PresentedViewController
+{
+    var doneButtonEnabled : Bool { get }
+    func cancelButtonWasPressed(sender: AnyObject)
+    func doneButtonWasPressed(sender: AnyObject)
```

Comment:
Following the name discussion, I'd greatly simplify the contents of the protocol:

```swift
protocol Confirmable {
    func cancel()
    func confirm()
}
```

Some reasons behind this:

- I'm not 100% sure about removing `doneButtonEnabled`, but I don't think it's needed in the protocol for KVO and we can default to `true`. If we must keep it there, I'd mark it as optional, and adjust the name.
- The content shouldn't know about `sender`. We're doing this so the "content" VCs can be agnostic about how they're being presented
- `ButtonWasPressed` is irrelevant information. We care about the action that needs to be done, not how we got there. As a counter example, we could decide that an iPad modal is canceled by tapping outside the form sheet, or add keyboard shortcuts to cancel/save.